vendor/bat-native-ads/src/bat/ads/internal/creatives/notification_ads/creative_notification_ads_database_table.cc

```diff
@@ -10,6 +10,7 @@
 
 #include "base/containers/contains.h"
 #include "base/functional/bind.h"
+#include "base/strings/string_split.h"
```

Comment:
not used, please let me know when you have fixed all warnings from clangd/clang-tidy